.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+#### Description
+
```

Comment:
Add:
The description should give a general overview of the goal of the PR.
Individual commit message should highlight important changes that people may want to know about year later when sorting thorough as commits are the changelog.

Reply:
Done.